tests/testthat/test-logisticregutils.R

```diff
@@ -0,0 +1,34 @@
+context("Logistic Regression: helper functions testing ")
+
+config <- list(
+  `graph.resolution` = dropdownInput('%Question.graph.resolution%' , '1x'),
+  `the.link` = dropdownInput('%Question.Link%' , 'logit'),
+  `model.name` = textInput('%Question.Model Name%', 'my_model'),
+  `used.weights` = checkboxInput('%Question.Use Weights%' , FALSE),
+  `Weight Vec` = dropdownInput('%Question.Weight Vec%'),
+  `X Vars` = listInput('%Question.X Vars%', c('Sepal.Length', 'Petal.Length')),
+  `Y Var` = dropdownInput('%Question.Y Var%', 'Species')
+)
+
+if (config$the.link == "complementary log-log"){
+  config$the.link <- "cloglog"
+}
+
+defaults <- list(
+  data = transform(iris,
+                   Species = as.factor(ifelse(Species != "setosa", "other", "setosa"))
+  )[,5:1]
```

Comment:
Rstudio automatically does that if I just hit "Enter". I was lazy and didn't do anything about it. I like this format better, since the second line aligns with function argument in the first line and it's clear this is part of the function call. However, I can go either way to keep it consistent. Changed